01PITT_INST-ULS_1_5/css/custom1.css

```diff
@@ -290,4 +290,17 @@ md-input-container label[translate="almaRequest.comment"]{position:relative;}
 	margin-bottom: 0; }
 
 #noContactRequestInstructions{display:none; background-color:pink; padding:5px;}
-#shipItRequestInstructions{display:none; background-color:pink; padding:5px;}
\ No newline at end of file
+#shipItRequestInstructions{display:none; background-color:pink; padding:5px;}
+
+/* SMS Notifications Preferences */
+.pink{animation: flash 2s forwards linear normal;}
```

Comment:
This didn't seem to work in our online demo.

Reply:
I tried a few different methods of implementing instant-feedback validation and had a tough time getting the field to light up and un-light in a user-friendly and reliable manner.  I abandoned in favor of a much simpler solution.  Displaying the validation error message only on submit.